pkg/cmd/cli/cmd/newapp.go

```diff
@@ -49,8 +49,8 @@ configuration, and a service will be connected to the first public port of the a
 components using the various existing flags or let new-app autodetect what kind of components
 you have provided.
 
-If you provide source code, you may need to run a build with 'start-build' after the
-application is created.`
+If you provide source code, a new build will be automatically triggered.
+You can use '%[1]s status' to check the progress.`
```

Comment:
need to update this to be replaced with `oc` like the top level `ocLongDesc` does. Will do after lunch.